xhprof_lib/utils/xhprof_runs.php

```diff
@@ -107,22 +107,22 @@ public static function getDbClass()
   * 
   *
 CREATE TABLE `details` (
```

Comment:
It won't, because when strict mode isn't enabled on MySQL server data will be truncated silently and SQL will be executed. By default strict mode is disabled. I'm not sure what setting is default for RDS on AWS.